yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/ActivityDetailsFactory.java

```diff
@@ -17,14 +17,11 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
-import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_CHARSET;
-import static com.yoti.api.client.spi.remote.call.YotiConstants.SYMMETRIC_CIPHER;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.*;
```

Comment:
This was done by the autoformatter in Intellij - I can put it back but if someone uses autoformat it will just go back to this.  @bucky-boy what's the general rule of thumb around using * for static imports?